v1/content.yaml

```diff
@@ -152,6 +152,7 @@ paths:
           required: false
       produces:
         - text/html; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/HTML/1.5.0"
```

Comment:
Move to 1.6.0

Reply:
@subbuss We're not ready to bump the expected version yet - this is just to fix the failing tests. We still want to be able to deploy RESTBase one time before we will be ready to for you to bump the version. So we will bump it in a separate one.

Reply:
ok